Jenkinsfile

```
@@ -105,12 +122,14 @@ pipeline {
 
                 stage('Test (Go)') {
                     steps {
-                        container('golang') {
-                            sh '.ci/scripts/distribution/build-go.sh'
-                        }
+                        timeout(time: shortTimeout, unit: 'MINUTES') {
```

```suggestion
                        timeout(time: longTimeout, unit: 'MINUTES') {
```

I would just standardize all verification stages to use longTimeout, just go give ourselves room as we add more tests.